tools/interactive/interactivetool_napari.xml

```diff
@@ -0,0 +1,47 @@
+<tool id="interactive_tool_napari" tool_type="interactive" name="napari" version="@VERSION@+galaxy0" profile="23.1">
+	<description>cell image analysis software</description>
+    <macros>
+        <token name="@VERSION@">0.4.18</token>
+    </macros>
+    <requirements>
+	    <container type="docker">registry.git.embl.de/grp-cbbcs/abcdesktop-apps/napari-galaxy:latest</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="Napari" requires_domain="True">
+            <port>5800</port>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code">
+    <![CDATA[
+        export HOME=\$PWD &&
+        ## Create a directory where the app user has access
+        mkdir -p ./output &&
+        chown 1000:1000 ./output/ &&
+
+
+        #if $infile:
+	    cp '$infile' ./$infile.display_name &&
+        #end if
+	/init
+
+    ]]>
+    </command>
+    <inputs>
+	<param name="infile" type="data" optional="true" format="jpg,png,tiff,bmp,gif,pcx,ppm,psd,pbm,pgm,eps" label="Images" />
+
+    </inputs>
+    
+    <outputs>
+	<data name="version" format="txt" label="${tool.name} on ${on_string}: version.txt" from_work_dir="version.txt"/>
+	<collection name="outputs" type="list" label="Napari outputs">
+            <discover_datasets pattern="__name_and_ext__" directory="output" />
+        </collection>
+    </outputs>
+    
+    <tests>
+    </tests>
+    
+    <help><![CDATA[
```

Comment:
Can you please add some help here. Explain what napari is and maybe add a citation?